static/javascripts/components/profile.jsx

```diff
@@ -49,65 +37,88 @@ var ProfilePage = React.createClass({
-                     <option value={this.state.ethnicity[0]}>{this.state.ethnicity[0]}</option>
-                     <option value={this.state.ethnicity[1]}>{this.state.ethnicity[1]}</option>
-                     <option value={this.state.ethnicity[2]}>{this.state.ethnicity[2]}</option>
-                     <option value={this.state.ethnicity[3]}>{this.state.ethnicity[3]}</option>
-                     <option value={this.state.ethnicity[4]}>{this.state.ethnicity[4]}</option>
-                     <option value={this.state.ethnicity[5]}>{this.state.ethnicity[5]}</option>
-                     <option value={this.state.ethnicity[6]}>{this.state.ethnicity[6]}</option>
-                   </select><br/>
-        Native Language: <select name="native_language">
-                           <option value={this.state.user_native_language}>{this.state.user_native_language}</option>
-                           <option value={this.state.native_language[0]}>{this.state.native_language[0]}</option>
-                           <option value={this.state.native_language[1]}>{this.state.native_language[1]}</option>
-                           <option value={this.state.native_language[2]}>{this.state.native_language[2]}</option>
-                           <option value={this.state.native_language[3]}>{this.state.native_language[3]}</option>
-                           <option value={this.state.native_language[4]}>{this.state.native_language[4]}</option>
-                           <option value={this.state.native_language[5]}>{this.state.native_language[5]}</option>
-                           <option value={this.state.native_language[6]}>{this.state.native_language[6]}</option>
-                           <option value={this.state.native_language[7]}>{this.state.native_language[7]}</option>
-                           <option value={this.state.native_language[8]}>{this.state.native_language[8]}</option>
-                           <option value={this.state.native_language[9]}>{this.state.native_language[9]}</option>
-                           <option value={this.state.native_language[10]}>{this.state.native_language[10]}</option>
-                           <option value={this.state.native_language[11]}>{this.state.native_language[11]}</option>
+    var edit_state = this.state.edit ? "edit mode: ON" : "edit mode: OFF";
+    if (edit_state == "edit mode: OFF") {
+      return(
+        <ul style={style}>
+          <p onClick={this.handleClick}>
+            Click to toggle. {edit_state}.
+          </p>
+          <li>Username: {this.state.username}</li>
+          <li>Affiliation: {this.state.user_affiliation}</li>
+          <li>Email: {this.state.email}</li>
+          <li>Birth Date: {this.state.dob}</li>
+          <li>Gender: {this.state.user_gender}</li>
+          <li>Ethnicity: {this.state.user_ethnicity}</li>
+          <li>Native Language: {this.state.user_native_language}</li>
+          <li>Academic Year: {this.state.user_status}</li>
```

Comment:
Would really appreciate ideas on how to separate this field out if the user is a faculty. Obviously, I can't put an if statement in the return.